packages/tauri/webpack.config.js

```diff
@@ -48,24 +49,29 @@ module.exports = {
     },
     plugins: [
         new EnvironmentPlugin({
+            PL_APP_NAME: name,
```

Comment:
These missing variables were breaking the app. I kept the defaults the same as PWA.